mmdet/datasets/dataset_wrappers.py

```diff
@@ -311,6 +312,8 @@ def __init__(self,
                  pipeline,
                  dynamic_scale=None,
                  skip_type_keys=None):
+        warnings.warn('dynamic_scale is deprecated. will be removed '
```

Comment:
If raising warning here, the _dynamic_scale function should not be removed.

Reply:
If this function is no longer supported, please raise an error and give some information, like "Please use another method..." or something.